modules/ept-frontend/client/admin/moderation/logs.controller.js

```diff
@@ -174,6 +174,9 @@ var ctrl = ['$rootScope', '$scope', '$location', '$timeout', '$anchorScroll', 'A
     { group: 'Thread Actions',
       value: 'threads.lockPoll',
       desc: 'Locked Thread Poll' },
+    { group: 'Thread Actions',
```

Comment:
There is actually another file that needs to be edited for the action to be actually logged in the moderation log see `/server/plugins/moderation_log/templates.js`. Also the enums for the moderation_action_type would have to be updated, see the latest migration `epochtalk-core-pg/migrations/sqls/20160421232530-user-notes-up.sql`

The mod log should only be updated if a moderator comes in and manually changes the threads owners, see users.update for example `epochtalk/modules/ept-users/routes/update.js` If a mod comes in and edits a user's account then it is logged to the mod log.

Sorry that this isn't well documented yet. I believe we will need permissions for the co-owners routes as well